awx/ui_next/src/screens/Host/Host.jsx

```diff
@@ -116,23 +101,25 @@ function Host({ i18n, setBreadcrumb }) {
         {showCardHeader && <RoutedTabs tabsArray={tabsArray} />}
         <Switch>
           <Redirect from="/hosts/:id" to="/hosts/:id/details" exact />
-          {host && [
-            <Route path="/hosts/:id/details" key="details">
-              <HostDetail host={host} />
-            </Route>,
-            <Route path="/hosts/:id/edit" key="edit">
-              <HostEdit host={host} />
-            </Route>,
-            <Route key="facts" path="/hosts/:id/facts">
-              <HostFacts host={host} />
-            </Route>,
-            <Route path="/hosts/:id/groups" key="groups">
-              <HostGroups host={host} />
-            </Route>,
-            <Route path="/hosts/:id/completed_jobs" key="completed-jobs">
-              <JobList defaultParams={{ job__hosts: host.id }} />
-            </Route>,
-          ]}
+
+          <Route path="/hosts/:id/details" key="details">
+            {isLoading ? <ContentLoading /> : <HostDetail host={host} />}
```

Comment:
I understand we are going to need to do some across-the-app kind of changes.

Is there a way we can use this `isLoading ? <ContentLoading /> : <Screen />` pattern always?  if there are some that we have to do this, and then sometimes you have to pass as prop, it's going to be make it more difficult to implement as well as when we add new views.

Reply:
Sometimes, if the child component does not make any api calls on its own I can just put the content loading like isLoading ? <contentLoading> : <screen> other times, if the child or grand child makes api calls I need to pass the loading state down so that it doesn’t flash the loading screen